app/src/main/java/com/vestrel00/daggerbutterknifemvp/ui/common/view/BaseFragment.java

```diff
@@ -62,9 +64,23 @@
     @Nullable
     private Unbinder viewUnbinder;
 
+    @SuppressWarnings("deprecation")
+    @Override
+    public void onAttach(Activity activity) {
+        if (Build.VERSION.SDK_INT < Build.VERSION_CODES.M) {
+            // Perform injection here before M, L (API 22) and below because onAttach(Context)
+            // is not yet available at L.
+            AndroidInjection.inject(this);
+        }
+        super.onAttach(activity);
+    }
+
     @Override
     public void onAttach(Context context) {
-        AndroidInjection.inject(this);
+        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.M) {
+            // Perform injection here for M (API 23) due to deprecation of onAttach(Activity).
+            AndroidInjection.inject(this);
```

Comment:
This build version checking is not necessary when using support Fragments. See the support setup: https://github.com/vestrel00/android-dagger-butterknife-mvp/pull/49/files#r133044925